src/backend/distributed/citus.sql

```diff
@@ -15,6 +15,20 @@ BEGIN
 END;
 $$;
 
+/*****************************************************************************
+ * Enable SSL to be secure by default
```

Comment:
change to `Enable SSL to encrypt the data by default`. To call it secure when MITM is possible gives probably the wrong signal.